brainscore/metrics/accuracy_delta.py

```diff
@@ -0,0 +1,90 @@
+import itertools
+
+import numpy as np
+
+from brainscore.metrics import Metric, Score
+import random
+from brainscore.metrics.ceiling import SplitHalvesConsistencyBaker
+
+
+class AccuracyDelta(Metric):
+
+    def __call__(self, source, target, image_types, isCeiling=False):
+
+        # if ceiling, then use already existing human delta code on two halves:
+        if isCeiling:
+            half_1_delta = self.get_human_delta(source, image_types)
+            half_2_delta = self.get_human_delta(target, image_types)
+            score = max((1 - ((np.abs(half_1_delta - half_2_delta)) / half_2_delta)), 0)
+            return score
+        else:
+            model_delta = self.get_model_delta(source, image_types)
+            scores = []
+
+            # calculate score over average of 100 sub splits of human delta
+            for i in range(100):
+                human_delta = self.get_human_delta(target, image_types)
+                score = max((1 - ((np.abs(human_delta - model_delta)) / human_delta)), 0)
+                scores.append(score)
+
+            # calculate mean:
+            score = np.mean(scores)
+
+            # calculate error
+            error = np.std(scores)
+
+            return Score([score, error], coords={'aggregation': ['center', 'error']}, dims=('aggregation',))
+
+    def extract_subjects(self, assembly):
+        return list(sorted(set(assembly['subject'].values)))
+
+    def get_human_delta(self, target, image_types):
+        # calculate human accuracies for [whole, condition]
+        condition_scores_human = []
+
+        # pull half of subjects. This is outside loop to make sure
+        # the same half of subjects are used in w and f conditions.
+        num_subjects = len(set(target["subject"].values))
+        half1_subjects = random.sample(range(1, num_subjects), num_subjects // 2)
```

Comment:
I don't like using un-seeded random generators because it makes the output non-deterministic and not perfectly reproducible. 

Instead use numpy's `RandomState` and seed: define `random_state = RandomState(0)` in the `__call__` method and pass the state to this method which can then call `random_state.choice`